src/qibolab/sweeper.py

```diff
@@ -0,0 +1,21 @@
+from dataclasses import dataclass
+from typing import Optional
+
+import numpy.typing as npt
+
+
+@dataclass
+class Sweeper:
+
+    parameter: str
```

Comment:
If this parameter has a finite domain (e.g. `["attenuation", "gain", "frequency"]`, consider making it an Enum instead.
```py
from enum import Enum, auto

class Kind(Enum):
    attenuation = auto()
    gain = auto()
    frequency = auto()
```

Reply:
I'm not really familiar with `Enum`. Here you are suggesting that `parameter` shoud not be a string but `Enum`, correct? In this case how would you exactly initialize a `Sweeper` with this syntax?

Reply:
```py
Sweeper(Kind.gain, ...)
```

In case you get a string from somewhere, e.g. a runcard, you can always do:
```py
Sweeper(Kind[my_kind_str], ...)
```

Reply:
Understood.
Thanks.

Reply:
Implemented I choose the name `Parameter` I could't think of anything else. I believe that it is slightly better than `Kind`.